src/components/Typography/Typography.style.ts

```diff
@@ -126,5 +152,14 @@ export const typographyWrapper =
       body03,
     };
 
-    return allStyles[type];
+    const textColorCategory = isInverted ? 'inverted' : 'light';
+    const textColor = `${textColorCategory}.${type}` as DotKeys<typeof textColorFigma>;
+
+    return css`
+      ${allStyles[variant]};
+      font-style: ${isItalic ? 'italic' : undefined};
+      font-weight: ${isBold ? 'bold' : undefined};
```

Comment:
should this be `bold` or some value from our global font-weight ones?

Reply:
this will implement `bold` or will not be defined at all, thus if there are global CSS will be applied